workers/unity/Packages/com.improbable.gdk.core/GameObjectRepresentation/GameObjectCreation/IEntityGameObjectCreator.cs

```diff
@@ -0,0 +1,10 @@
+﻿using UnityEngine;
+
+namespace Improbable.Gdk.Core.GameObjectRepresentation
+{
+    public interface IEntityGameObjectCreator
```

Comment:
Is there a doc on the design decision that the game object creation now has to go through an interface?

Reply:
This was a change of plans from the design doc I discussed with Sami and Matt after realizing the solution I had planned in the doc wasn't good or even possible because it put something relying on the Transform component, from a different feature module, in the monobehaviour feature module (ie in core for now really since it's not a proper module now).